recipes/compilers/meta.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: compilers-metapackages
+  version: 1.0.0
+
+build:
+  number: 0
+
+outputs:
+  - name: c-compiler
+    requirements:
+      run:
+        - {{ compiler('c') }}
+    test:
+      commands:
+        - $CC --help  # [not win]
+  - name: cxx-compiler
+    requirements:
+      run:
+        - {{ compiler('cxx') }}
+    test:
+      commands:
+        - $CXX --help  # [not win]
+  - name: fortran-compiler
+    requirements:
+      run:
+        - {{ compiler('fortran') }}
+    test:
+      commands:
+        - $FC --help  # [not win]
+  - name: compilers
+    requirements:
+      run:
+        - {{ pin_subpackage('c-compiler', exact=True) }}
+        - {{ pin_subpackage('cxx-compiler', exact=True) }}
+        - {{ pin_subpackage('fortran-compiler', exact=True) }}
+    test:
+      commands:
+        - $CC --help  # [not win]
+        - $CXX --help  # [not win]
+        - $FC --help  # [not win]
+
+about:
+  summary: Compiler metapackages
+  home: https://conda-forge.org
+  license: BSD
```

Comment:
A longer description would be nice here

Reply:
Aye, I know, can you add/suggest one? I'm new to this so don't really know what to include.

Reply:
Something like

```yaml
description: |
    This package is a generic way to obtain the compilers for your system that conda-forge used to compile 
    its ecosystem. These compilers are, therefore, guaranteed to be ABI compatible with the conda packages
    you have installed. 

    These compiler metapackages are a convenience ONLY for users.  Do NOT use these packages as a
    build or host dependencies in other recipes.  Use the compiler Jinja template function instead. 
    For C++ for example, use compiler('cxx') as usual.
```